airflow/kubernetes/pod_launcher.py

```diff
@@ -124,14 +124,14 @@ def _monitor_pod(self, pod: V1Pod, get_logs: bool) -> Tuple[State, Optional[str]
         if get_logs:
             logs = self.read_pod_logs(pod)
             for line in logs:
-                self.log.info(line)
+                self.log.info("Log: %s", line.decode().rstrip())
```

Comment:
Are these the logs that show up in task logs? If so then we probably don't want this change

Reply:
We should decode output. Otherwise it will be displayed in an ugly way.
```
[2019-08-29 11:06:04,975] {pod_launcher.py:106} INFO - b'\n'
[2019-08-29 11:06:04,976] {pod_launcher.py:106} INFO - b'Loading DB routines from perl5db.pl version 1.55\n'
[2019-08-29 11:06:04,977] {pod_launcher.py:106} INFO - b'Editor support available.\n'
[2019-08-29 11:06:04,977] {pod_launcher.py:106} INFO - b'\n'
[2019-08-29 11:06:04,978] {pod_launcher.py:106} INFO - b"Enter h or 'h h' for help, or 'man perldebug' for more help.\n"
[2019-08-29 11:06:04,979] {pod_launcher.py:106} INFO - b'\n'
[2019-08-29 11:06:04,979] {pod_launcher.py:106} INFO - b'main::(-e:1):\t0\n'
[2019-08-29 11:06:04,981] {pod_launcher.py:106} INFO - b'  DB<1> '
```
I removed prefix "Log:"